Sources/CartonHelpers/DefaultToolchain.swift

```diff
@@ -12,4 +12,4 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-public let defaultToolchainVersion = "wasm-5.8.0-RELEASE"
+public let defaultToolchainVersion = "wasm-5.9-SNAPSHOT-2023-08-01-a"
```

Comment:
Please use 5.9-RELEASE for the toolchain version.
<sub>Even though it seems fine, we don't want to use a pre-release toolchain for a production carton release.</sub>